lib/OpenQA/WebAPI/Controller/API/V1/Comment.pm

```diff
@@ -191,12 +193,22 @@ sub create_many ($self) {
         push @failed, {job_id => $job_id} if $@;
     }
 
+    if ($wanna_restart && $wanna_restart == 1) {
+        for my $job_id (@$job_ids) {
+            my ($res, $jobs, $auto, $single_job_id, $dup_route);
+            my %args = (jobs => $job_id);
+            $self->param('jobid', $job_id);
+            eval { ($res, $jobs, $auto, $single_job_id, $dup_route) = $self->restart_job(\%args); };
```

Comment:
The `_restart` method already supports multiple jobs as parameters.
The documentation for the `/restart` route says "Restart job(s)."

Calling this for every single job and validating the same request parameters again and again is not necessary.
Also you are passing something in `%args` but you never read that in the function.

And there is already one function that restarts one job, that could also be called, instead of calling the whole `_restart` thing:
```
my $res = OpenQA::Resource::Jobs::job_restart($jobs, @params);
```

Reply:
@Martchus and that's actually also what I meant earlier today: `_restart` is a complete endpoint:
```
sub restart ($self) { $self->_restart }
```
making a helper that implements a complete endpoint, doesn't make much sense to me.